### ISSUE TITLE
Latte fails with segmentation faul

### ISSUE THREAD

**Issue by shackra** [flakas/Latte#15]
```
may 03 13:06:32 sanson-gnu systemd-coredump[6984]: Process 6963 (latte) of user 1000 dumped core.
                                                   
                                                   Stack trace of thread 6963:
                                                   #0  0x00007f9d11f756a0 XDefaultRootWindow (libX11.so)
                                                   #1  0x00007f9d14d2e1c8 ffi_call_unix64 (libffi.so.6)
                                                   #2  0x00007f9d14d2dc2a ffi_call (libffi.so.6)
                                                   #3  0x00007f9d14f41216 _ctypes_callproc (/usr/lib/python2.7/lib-dynload/_ctypes.so)
                                                   #4  0x00007f9d14f3ace0 n/a (/usr/lib/python2.7/lib-dynload/_ctypes.so)
                                                   #5  0x00007f9d1788dd83 PyObject_Call (libpython2.7.so.1.0)
                                                   #6  0x00007f9d1782dfde PyEval_EvalFrameEx (libpython2.7.so.1.0)
                                                   #7  0x00007f9d178c7746 PyEval_EvalCodeEx (libpython2.7.so.1.0)
                                                   #8  0x00007f9d17875c48 function_call.lto_priv.225 (libpython2.7.so.1.0)
                                                   #9  0x00007f9d1788dd83 PyObject_Call (libpython2.7.so.1.0)
                                                   #10 0x00007f9d178865ff instancemethod_call.lto_priv.140 (libpython2.7.so.1.0)
                                                   #11 0x00007f9d1788dd83 PyObject_Call (libpython2.7.so.1.0)
                                                   #12 0x00007f9d1786abe8 slot_tp_init (libpython2.7.so.1.0)
                                                   #13 0x00007f9d178940e5 type_call.lto_priv.51 (libpython2.7.so.1.0)
                                                   #14 0x00007f9d1788dd83 PyObject_Call (libpython2.7.so.1.0)
                                                   #15 0x00007f9d1782dfde PyEval_EvalFrameEx (libpython2.7.so.1.0)
                                                   #16 0x00007f9d178c7746 PyEval_EvalCodeEx (libpython2.7.so.1.0)
                                                   #17 0x00007f9d17875c48 function_call.lto_priv.225 (libpython2.7.so.1.0)
                                                   #18 0x00007f9d1788dd83 PyObject_Call (libpython2.7.so.1.0)
                                                   #19 0x00007f9d178865ff instancemethod_call.lto_priv.140 (libpython2.7.so.1.0)
```

I have LibX11 available:

```
$ whereis libX11.so                                                                                                                
libX11: /usr/lib/libX11.so
```

**Comment by flakas** [comment]
I think this should have been fixed with the latest `4.0` release.
Please reopen the issue if the problem persists.

**Comment by shackra** [comment]
It seems that it works now, thanks!
